cfme/infrastructure/cluster.py

```diff
@@ -137,6 +148,15 @@ def am_i_here(self):
         return match_page(summary="{} (Summary)".format(self.obj.name))
 
 
+@navigator.register(Cluster, 'Timelines')
+class Timelines(CFMENavigateStep):
```

Comment:
```
from cfme.infrastructure.cluster import Cluster
from utils.providers import get_crud
from utils.appliance.implementations.ui import navigate_to

p = get_crud('vsphere55-old')
c = Cluster('test_cluster in test_datacenter', p)
navigate_to(c, "Details")

# Nothing happens because navmazing assumes that we are already on Timelines page so the is_displayed for the Timelines is incorrect
navigate_to(c, "Timelines")
```